datastore/redis.go

```diff
@@ -216,8 +216,17 @@ func (r *RedisCache) GetKnownValidators() (map[uint64]boostTypes.PubkeyHex, erro
 	return validators, nil
 }
 
+func (r *RedisCache) SetMultiKnownValidator(indexPkMap map[uint64]boostTypes.PubkeyHex) error {
+	values := []string{}
+	for proposerIndex, publickeyHex := range indexPkMap {
+		values = append(values, strconv.FormatUint(proposerIndex, 10), PubkeyHexToLowerStr(publickeyHex))
+	}
+
+	return r.client.HSet(context.Background(), r.keyKnownValidators, values).Err()
```

Comment:
I'm actually seeing this error with Redis 7.0.3:

```ERR wrong number of arguments for 'hset' command```

Let's switch back to HMSET for now?